[反馈]: Request processing failed: Google API returned error: 400 INVALID_ARGUMENT

### 部署平台

None

### 版本号

v0.5.1

### 客户端/工具

opencode 

### 问题/请求描述

curl可以正常调用api, opencode下会报INVALID_ARGUMENT,  opencode 配置
```
"provider": {
    "aistudio": {
       "name": "Google",
       "models": {
          "gemini-3-pro-preview": {
            "name": "Gemini 3 pro preview(aistudio)",
            "limit": {
              "context": 1048576,
              "output": 65535
            },
            "modalities": {
              "input": [
                "text",
                "image",
                "pdf"
              ],
              "output": [
                "text"
              ]
            },
            "variants": {
              "low": {
                "thinkingLevel": "low"
              },
              "high": {
                "thinkingLevel": "high"
              }
            }

          }
        }
      }
    }

```

### 日志信息（可选）

```text
2026-01-26T11:03:52.146Z [ProxySystem] - [Browser] Received request: POST /v1beta/models/gemini-3-pro-preview:streamGenerateContent
[INFO] 2026-01-26T11:03:52.146Z [ProxySystem] - [Browser] Browser received request
[INFO] 2026-01-26T11:03:52.146Z [ProxySystem] - [Browser] Executing request: POST /v1beta/models/gemini-3-pro-preview:streamGenerateContent
[ERROR] 2026-01-26T11:03:52.799Z [ProxySystem] - [Adapter] Received error from browser, will trigger switching logic. Status code: 400, message: Proxy browser error: Google API returned error: 400 INVALID_ARGUMENT {"error":{"code":400,"message":"Request contains an invalid argument.","status":"INVALID_ARGUMENT"}}
[WARN] 2026-01-26T11:03:52.800Z [ProxySystem] - ⚠️ [Auth] Request failed - failure count: 3/3 (Current account index: 0)
[WARN] 2026-01-26T11:03:52.800Z [ProxySystem] - 🔴 [Auth] Failure threshold reached (3/3)! Preparing to switch account...
[INFO] 2026-01-26T11:03:52.800Z [ProxySystem] - ==================================================
[INFO] 2026-01-26T11:03:52.801Z [ProxySystem] - 🔄 [Auth] Single account mode: Rotation threshold reached, performing in-place restart...
[INFO] 2026-01-26T11:03:52.801Z [ProxySystem] -    • Target account: #0
[INFO] 2026-01-26T11:03:52.801Z [ProxySystem] - ==================================================
[INFO] 2026-01-26T11:03:52.803Z [ProxySystem] - [Browser] ❌ Request processing failed: Google API returned error: 400 INVALID_ARGUMENT {"error":{"code":400,"message":"Request contains an invalid argument.","status":"INVALID_ARGUMENT"}}
[INFO] 2026-01-26T11:03:52.804Z [ProxySystem] - [Browser] Sent 'error' information back to server
[INFO] 2026-01-26T11:03:52.976Z [ProxySystem] - [Auth Update] 💾 Successfully updated auth credentials for account #0
[INFO] 2026-01-26T11:03:52.977Z [ProxySystem] - [Browser] Stopped background tasks (Scavenger) for old page.
[INFO] 2026-01-26T11:03:52.977Z [ProxySystem] - [Browser] Closing old API browser context...
[INFO] 2026-01-26T11:03:52.983Z [ProxySystem] - [Browser] Old API context closed.
```

--- a/src/core/FormatConverter.js
+++ b/src/core/FormatConverter.js
@@ -514,11 +514,19 @@ class FormatConverter {
             for (const key of Object.keys(obj)) {
                 // 1. Filter out unsupported fields using a blacklist approach
                 // This allows potentially valid fields to pass through while blocking known problematic ones
-                const unsupportedKeys = ["$schema", "additionalProperties", "ref", "$ref"];
+                const unsupportedKeys = [
+                    "$schema",
+                    "additionalProperties",
+                    "ref",
+                    "$ref",
+                    "propertyNames",
+                    "patternProperties",
+                    "unevaluatedProperties",
+                ];
 
                 if (isResponseSchema) {
                     // For Structured Outputs: stricter filtering of metadata that causes 400 errors
-                    unsupportedKeys.push("title", "default", "examples", "$defs", "id", "patternProperties");
+                    unsupportedKeys.push("title", "default", "examples", "$defs", "id");
                 }
 
                 if (unsupportedKeys.includes(key)) {

